catalog/ui/search-ui/standard/src/assembly/images.xml

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser General Public License as published by the Free Software Foundation, either
+ * version 3 of the License, or any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ *
+ **/
+-->
+<assembly
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xmlns="http://maven.apache.org/plugins/maven-assembly-plugin/assembly/1.1.2"
+        xsi:schemaLocation="
+      http://maven.apache.org/plugins/maven-assembly-plugin/assembly/1.1.2
+      http://maven.apache.org/xsd/assembly-1.1.2.xsd">
+    <id>images</id>
+    <formats>
+          <format>zip</format>
+    </formats>
+    <fileSets>
+        <fileSet>
+             <fileMode>0644</fileMode>
+             <directoryMode>0755</directoryMode>
+             <directory>src/main/webapp/images</directory>
+             <includes>
+                   <include>**</include>
+            </includes>
+            <outputDirectory></outputDirectory>
```

Comment:
Thanks for fixing it & sorry for commenting in the wrong place! I meant to comment on the overall PR, but accidentally commented under Vina's comment for an unrelated file.